i18n/update_translations.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
```

Comment:
Yes with `nodejs`. The benefit of javascript is, that we don't need to introduce a third programming language (namely python).